ops/tag-service/tag-tool.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import subprocess
+import re
+import semver
+
+SERVICES  = [
+    'ci-builder',
+    'ci-builder-rust',
+    'chain-mon',
+    'indexer',
+    'op-node',
+    'op-batcher',
+    'op-challenger',
+    'op-dispute-mon',
+    'op-proposer',
+    'op-ufm',
+    'proxyd',
+    'op-heartbeat',
+    'ufm-metamask',
+    'op-contracts',
+    'full', # special case for tagging op-node, op-batcher, and op-proposer together
+]
+VERSION_PATTERN = '^{service}/v\\d+\\.\\d+\\.\\d+(-rc\\.\\d+)?$'
+GIT_TAG_COMMAND = 'git tag -a {tag} -m "{message}"'
+GIT_PUSH_COMMAND = 'git push origin {tag}'
+
+def new_tag(service, version, bump):
+    if bump == 'major':
+        bumped = version.bump_major()
+    elif bump == 'minor':
+        bumped = version.bump_minor()
+    elif bump == 'patch':
+        bumped = version.bump_patch()
+    elif bump == 'prerelease':
+        bumped = version.bump_prerelease()
+    elif bump == 'finalize-prerelease':
+        bumped = version.finalize_version()
+    else:
+        raise Exception('Invalid bump type: {}'.format(bump))
+    return f'{service}/v{bumped}'
+
+def latest_version(service):
+    # Get the list of tags from the git repository.
+    tags = subprocess.run(['git', 'tag', '--list'], capture_output=True, check=True) \
```

Comment:
```suggestion
    tags = subprocess.run(['git', 'tag', '--list', f'{service}/v*'], capture_output=True, check=True) \
```
can already filter here